data/dev-rakeshshaw.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Rakesh Shaw",
+  "type": "personal",
+  "bio": "👋Upcoming **Dev**👋. Success requires persistent practice, guided by patience."
```

Please add a missing comma `,` here.

```suggestion
  "bio": "👋Upcoming **Dev**👋. Success requires persistent practice, guided by patience.",
```

**Pro Tip:** You can commit the requested changes directly using the <kbd>Commit suggestion</kbd> button above.

---

Thanks for the suggestion, Bro.